modules/core/src/v2/coins/eos.ts

```diff
@@ -541,7 +542,7 @@ export class Eos extends BaseCoin {
         rpc: new NoopJsonRpc(),
         signatureProvider: new NoopSignatureProvider(),
         chainId: self.getChainId(),
-        textDecoder: new TextDecoder(),
+        textDecoder: new EosTextDecoder(),
```

Comment:
is this function also used by other services like TAT or platform-api?

Reply:
@shihao-guo did a code search in bitgo-ms and it's used primarily for `send` and `stake` transactions.
but staking is only supported for cspr and stx.